qxf2_scheduler/qxf2_scheduler.py

```diff
@@ -168,22 +212,14 @@ def get_free_slots(busy_slots, day_start, day_end):
         free_slots.append(day_start)
     for busy_slot in busy_slots:
         if busy_slot['end'] < day_start:
-            if busy_slots[-1]==busy_slot:
-                if len(free_slots) == 0:
```

Comment:
Why did this change? Or is it just a side-effect of not having merged master yet?